Nodes/SerialIO.c

```diff
@@ -370,16 +379,16 @@ char sergettkn(void) {
 
 void eka(char tecken) {
```

Comment:
At some point I want to get rid of the "eka" functions and just use SendString(). But I guess we're not quite there yet.